src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -178,7 +187,14 @@ public static TypedData ToRpc(this object value, ILogger logger)
 
                     request.Body.Position = 0;
                     http.Body = body.ToRpc(logger);
-                    http.RawBody = rawBody.ToRpc(logger);
+                    if (Capabilities.IsCapabilityEnabled(ExposedCapabilities.RawHttpBodyBytes))
```

Comment:
Update this to check for existence of this capability i.e
```
if (!string.IsNullOrEmpty(capabilities.GetCapabilityState(LanguageWorkerConstants.RawHttpBodyBytes)))
```

Reply:
Will address when opening a separate PR for adding raw body bytes capability.